2_process/src/reclassify_land_cover.R

```diff
@@ -0,0 +1,40 @@
+reclassify_land_cover <- function(land_cover_df,
```

Comment:
I think this same function shows up as new in the FORESCE PR. I'll look at this more closely after that PR is merged into USGS-R/main and USGS-R/main is merged into this branch.